src/components/Dashboard.js

```diff
@@ -0,0 +1,254 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+const Dashboard = () => {
+  const [service, setService] = useState('');
+  const [externalId, setExternalId] = useState(uuidv4());
+  const [serviceName, setServiceName] = useState('');
+  const [roleArn, setRoleArn] = useState('');
+  const [externalIdInput, setExternalIdInput] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleServiceChange = (e) => {
+    setService(e.target.value);
+  };
+
+  const handleServiceNameChange = (e) => {
+    setServiceName(e.target.value);
+  };
+
+  const handleRoleArnChange = (e) => {
+    setRoleArn(e.target.value);
+  };
+
+  const handleExternalIdInputChange = (e) => {
+    setExternalIdInput(e.target.value);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const data = { service_name: serviceName, role_arn: roleArn, external_id: externalIdInput };
+    
+    try {
+      const response = await fetch('http://localhost:5000/create_role', {
+        method: 'POST',
```

Comment:
same here , may be we can try other approach to get role details @DigiReaper  any feedback is appreciated

Reply:
yeah same, we can't hard code the localhost url